mubench.reviewsite/tests/TagsTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+require_once "DatabaseTestCase.php";
+
+use MuBench\ReviewSite\Controller\TagController;
+
+class TagTest extends DatabaseTestCase
+{
+
+    private $tagController;
+
+    function setUp()
+    {
+        parent::setUp();
+        $this->tagController = new TagController($this->db, $this->logger);
+        $this->db->table('tags')->insert(['name' => 'test1']);
+        $this->db->table('tags')->insert(['name' => 'test2']);
+        $this->db->table('tags')->insert(['name' => 'test3']);
+    }
+
+    function test_get_all_tags()
+    {
+        $tags = $this->db->getAllTags();
+        $expected_tags = [
+            ['id' => 1, 'name' => 'test1'],
+            ['id' => 2, 'name' => 'test2'],
+            ['id' => 3, 'name' => 'test3']
+        ];
+        self::assertEquals($expected_tags, $tags);
+    }
+
+    function test_save_misuse_tags()
+    {
+        $misuse_tag1 = [
+            "tag" => 'test1',
+            "exp" => 1,
+            "detector" => 1,
+            "project" => "-project-",
+            "version" => "-version-",
+            "misuse" => "-misuse-"
+        ];
+        $misuse_tag2 = [
```

Comment:
Why is it important to add a second tag here? If it is, we should have a test for adding a single tag and a second for adding two tags, because adding a single tag is definitely a use case.